sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedIngestionSettings.cs

```diff
@@ -46,11 +46,10 @@ internal DataFeedIngestionSettings(DataFeedDetail dataFeedDetail)
         /// </summary>
         public int? DataSourceRequestConcurrency { get; set; }
 
-        // TODO: provide link to documentation in IngestionRetryDelay.
-
         /// <summary>
         /// The minimum delay between two consecutive retry attempts, in case data ingestion
         /// fails. If not specified, the service's behavior depends on the data feed's granularity.
+        /// See the <see href="https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor/how-tos/onboard-your-data#avoid-loading-partial-data">documentation></see> for details.
```

Comment:
could you remove locale from the link?

Reply:
thanks - I always forget that - we need an analyzer for this :)

Reply:
me too. I think eng services added a check in the CI for that. OR they are planning on it